internal/cmd/common/flags.go

```diff
@@ -264,8 +265,10 @@ func HandleAttributeFlags(c *base.Command, suffix, fullField string, sepFields [
 		// First, perform any needed parsing if we are given the type
 		switch field.Name {
 		case "num-" + suffix:
+			if field.Value == nil {
+				return fmt.Errorf("num-%s flag does not support key-only values", suffix)
```

Comment:
"key-only values" sounds odd to me, perhaps something like "num-%s flag requires a value"?

Reply:
Fixed!